metron-interface/metron-alerts/pom.xml

```diff
@@ -130,4 +146,44 @@
             </plugin>
         </plugins>
     </build>
+  </profile>
+
+    <profile>
+      <id>e2e</id>
+      <build>
+        <plugins>
+          <plugin>
```

Comment:
should we document with a comment what these plugins are for?

Reply:
Once we get closer to consensus on what this is going to look like, I promise I will add extensive documentation.